slices/erlang-os-mon.yaml

```diff
@@ -0,0 +1,24 @@
+package: erlang-os-mon
+
+essential:
+  - erlang-os-mon_copyright
+
+slices:
+  bins:
+    essential:
+      - erlang-os-mon_libs
+    contents:
+      /usr/lib/erlang/lib/os_mon-*/priv/**:
```

Comment:
in some of the above, you're not including the `priv` folder. Is it forgotten or just not needed?
Also, if these are bins, can they be tested?

Reply:
e.g. erlang-inets has